tools/RELEASE.md

```diff
@@ -5,8 +5,8 @@ when no more blocking issues needs to be fixed a final tag is created
 and the final release is rolled.
 
 confluent-kafka-python uses semver versioning and loosely follows
-librdkafka's version, e.g. v0.11.4 for the final release and
-v0.11.4rc3 for the 3rd v0.11.4 release candidate.
+librdkafka's version, e.g. v0.11.5 for the final release and
```

Comment:
no need to update this, these are just examples.

Reply:
is there any problem with doing so?